.circleci/config.yml

```diff
@@ -62,11 +62,10 @@ jobs:
       SOLR_ADMIN_USER: admin
       SOLR_ADMIN_PASSWORD: admin
       SOLR_COLLECTION_NAME: hydra-test
-      SOLR_CONFIGSET_NAME: hyku-1
+      SOLR_CONFIGSET_NAME: hyku
```

Comment:
This may need to be hyku-1, I remember something about this, but my memory is a bit foggy.

Reply:
Does it matter since it is circleci?

Reply:
going to leave as-is so we are closer to Hyku and we don't use this file